mne/io/ctf/info.py

```diff
@@ -73,7 +73,7 @@ def _pick_isotrak_and_hpi_coils(res4, coils, t):
 
 def _convert_time(date_str, time_str):
     """Convert date and time strings to float time."""
-    for fmt in ("%d/%m/%Y", "%d-%b-%Y", "%a, %b %d, %Y"):
+    for fmt in ("%Y/%m/%d", "%d/%m/%Y", "%d-%b-%Y", "%a, %b %d, %Y"):
```

Comment:
can you put it at the end to limit the risk of a change of behavior?

Reply:
sure!